### ISSUE TITLE
Error with swift 5.4? 

### ISSUE THREAD

**Issue by bourvill** [WeTransfer/GitBuddy#68]
Hello,

Before 5.4 it's ok. 
Now I try to use Gitbuddy with swift 5.4 I got trace trap

```[1]    29091 trace trap```

I try to rebuild from source and run this 
```
swift run --package-path /Volumes/perso/GitBuddy/ GitBuddy changelog -s 1.1.1
```
And got 
```
Swift/ContiguousArrayBuffer.swift:580: Fatal error: Index out of range
[1]    29091 trace trap  swift run --package-path /Volumes/perso/GitBuddy/ GitBuddy changelog -s 1.1.1
```

**Comment by aboedo** [comment]
Seems like this is still valid with Swift 5.5.2. Any updates?

**Comment by AvdLee** [comment]
I'm about to create a new release, could you try with that one? I tried it locally and it works here

**Comment by aboedo** [comment]
same result... but it seems like there might be something off with the release? 
running
```bash
> gitbuddy --version
```
outputs 
```
3.1.1
```
even after upgrading. Always through mint. I tried on a clean mac and one that had 3.1.1 installed before. 

<img width="601" alt="image" src="https://user-images.githubusercontent.com/3922667/151063430-3f5407f3-83a7-408b-86ba-69cb2863af78.png">

Thoughts? I haven't tried manual install this time around

**Comment by aboedo** [comment]
yeah the tagged 4.0.0 version has 3.1.1 in the code https://github.com/WeTransfer/GitBuddy/blob/4.0.0/Sources/GitBuddyCore/Commands/GitBuddy.swift, can't tell if that's the only thing that's outdated in the tag, though

**Comment by aboedo** [comment]
feels like the tag might have been made in the wrong commit? https://github.com/WeTransfer/GitBuddy/compare/4.0.0...master

**Comment by AvdLee** [comment]
Ah, interesting! I'm pretty sure the release is made from the right commit, but I'll make sure to update the version number accordingly.

**Comment by AvdLee** [comment]
@aboedo I just did another try locally and it just works. Can you try running:

```
$ gitbuddy changelog -s 1.1.1 --verbose
```

Note: `--verbose` won't work if you use `--package-path` since the verbose flag is taken away by `swift`. If building from source, you want enable verbosity hardcoded.
